dev/com.ibm.jbatch.internal/build.properties

```diff
@@ -0,0 +1,13 @@
+###############################################################################
```

Comment:
What is the point of this file?

Reply:
Carried over the build config files from jbatch.container when I made the project, but it seems this isn't needed so I'll remove it.